src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -11,8 +11,9 @@ define([
     'sulucontent/model/content',
     'sulucontent/components/content/preview/main',
     'sulucontent/components/copy-locale-overlay/main',
+    'sulusecurity/services/security-checker',
     'config'
-], function(Content, Preview, CopyLocale, Config) {
+], function(Content, Preview, CopyLocale, SecurityChecker, Config) {
```

Comment:
hmm we should define how we write injected services ... i would write `securityChecker` because its an instance and not a constructor ...